pages/api/mail/index.ts

```diff
@@ -0,0 +1,111 @@
+import { ResponseUtil } from "@utils/responseUtil";
+import type { NextApiRequest, NextApiResponse } from "next";
+import send from "services/nodemailer/mail";
+import findEmails from "@database/mail";
+
+/**
+ * Handles the request to /api/mail (POST)
+ * @param req request object
+ * @param res response object
+ * @returns Promise<void> and sends json object back to the client (type: ResponseObject)
+ */
+export default async function mailHandler(
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> {
+    if (req.method == "POST" && req.body.key == process.env.AWS_LAMBDA_KEY) {
+        // AWS_LAMBDA_SECRET
+        const threeHours = 3;
+        const fortyEightHours = 48;
```

Comment:
nit: Variable names should be related to what those values contain and not the value itself (ex. `firstIntervalHours`). If we decide to change the value it shouldn't impact the variable name.